grumphp.yml

```diff
@@ -18,8 +18,10 @@ grumphp:
       run_on: ['web/modules/custom', 'web/themes/custom']
     yaml_lint: ~
     json_lint: ~
-    phpunit:
-      testsuite: unit
+# Enable after there are tests in web/modules/custom.
```

Comment:
Maybe we should think about some really light module in `web/modules/custom` to showcase the testing? And this should probably go to the testing section of the readme.